enterprise/server/tasksize/tasksize.go

```diff
@@ -2,13 +2,20 @@ package tasksize
 
 import (
 	"log"
+	"strconv"
+	"strings"
 
 	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
 	scpb "github.com/buildbuddy-io/buildbuddy/proto/scheduler"
 )
 
 const (
 	testSizeEnvVar = "TEST_SIZE"
+	// A BuildBuddy Compute Unit is defined as 1 cpu and 2.5GB of memory.
+	estimatedComputeUnitsPropertyKey = "EstimatedComputeUnits"
+	maxComputeUnits                  = 8         // We can't schedule tasks larger than 8 compute units.
```

Comment:
since we already compute the CPU max by looking at the machine -- can we remove this max?

Reply:
Done - also made the scheduler return an error if trying to schedule a task larger than any of the nodes can handle.